io.github.am2rcommunitydevelopers.AM2RLauncher.yaml

```diff
@@ -0,0 +1,134 @@
+app-id: io.github.am2rcommunitydevelopers.AM2RLauncher
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+  - org.freedesktop.Sdk.Extension.openjdk11
+  - org.gnome.Sdk.Compat.i386
+
+finish-args:
+# TODO: crashes with wayland on filepicker and font being whacky. Likely Eto bug, test and report there.
+- --socket=x11
+- --share=ipc
+
+# Network needed for updates, pulseaudio for sound
+- --share=network
+- --socket=pulseaudio
+
+# Home for getting the 1.1 zip, config because of profiles having their saves there, apktool for apk creation
+- --filesystem=home:ro
+- --filesystem=~/.config/
+- --filesystem=xdg-data/apktool/:create
+
+# libappindicator stuff
+- --talk-name=com.canonical.indicator.application
+- --talk-name=org.freedesktop.Notifications
+- --talk-name=org.kde.StatusNotifierWatcher
+
+# Fuse / Appimage related
+- --talk-name=org.freedesktop.Flatpak.*
+- --device=all
+- --filesystem=/tmp
+
+# Java and x32 stuff
+- --env=JAVA_HOME=/app/jre
+- --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+- --env=XDG_CONFIG_DIRS=/etc/xdg:/usr/lib/x86_64-linux-gnu/GL:/usr/lib/i386-linux-gnu/GL
+
+add-extensions:
+  org.gnome.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '42'
+
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    version: '21.08'
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d;vulkan/implicit_layer.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+
+command: AM2RLauncher
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: libappindicator
```

Comment:
```suggestion
```

Reply:
From the [instructions here](https://github.com/flathub/shared-modules) I thought I was supposed to provide a name, didn't know it wasn't necessary.